parachain/scripts/generate-release-notes.sh

```diff
@@ -211,6 +211,8 @@ if is_omni_executor_release; then
   WORKER_RUSTC_VERSION=$(cd tee-worker/omni-executor && rustc --version)
   docker run --rm --entrypoint gramine-sgx-sigstruct-view -v /var/run/aesmd:/var/run/aesmd litentry/omni-executor:$OMNI_EXECUTOR_DOCKER_TAG omni-executor.sig > enclave-sigstruct.txt
   SIGSTRUCT=$(<enclave-sigstruct.txt)
+  # Format the sigstruct output with proper indentation
+  FORMATTED_SIGSTRUCT=$(echo "$SIGSTRUCT" | sed 's/^/    /')
```

Comment:
Would you make `SIGSTRUCT` aligned with other items like 

```
client version               : 0.1.0
client name                  : executor-worker
rustc                        : rustc 1.87.0 (17067e9ac 2025-05-09)
docker image                 : litentry/omni-executor:v0.9.25-01
```
?

Not sure if that's difficult

Reply:
In the PR we add a extra line and also add some indentation so it looks like a sub section.

Reply:
I'm not sure if I missing it, but I don't seem to find the extra line 😂